website/docs/pushdown.md

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_position: 1
+---
+
+# Query pushdown
+
+**Query pushdown** is the method of optimizing a query by reducing the amount of data read and processed.
+It saves memory space, network bandwidth, and reduces the query execution time by not prefetching
+unnecessary data to the database management system.
+
+:::info
+You can read more about query pushdown in our [blog post](https://blog.ferretdb.io/ferretdb-fetches-data-query-pushdown/)!
+:::
+
+## Why do we need query pushdown
+
+As FerretDB mission is to become true open-source MongoDB-compatible database, it's important
+to handle all operations, comparisons, data types, and commands in the same fashion as MongoDB.
```

Comment:
I think technical documentation (at that part at least) there should be more neutral and "boring". That sounds like an exiting blog post.

We mostly need to document three things:
1. What is pusdown.
2. What we support.
3. What are caveats.

Reply:
Yeah, what you already wrote below